airbyte-integrations/connectors/destination-s3-data-lake/gradle.properties

```diff
@@ -1,2 +1,2 @@
 testExecutionConcurrency=-1
-cdkVersion=0.602
+cdkVersion=local
```

Comment:
didn't end up needing this, I just wanted to have an easier time with the debugger

(but it also proves that the connector is actually in pretty good shape - no major breakages from all the CDK updates)